### PR TITLE
Changelog update for upcoming 10.0.9

### PR DESCRIPTION
**Pull request by PVince81**
Update changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,17 +5,56 @@ All notable changes to this project will be documented in this file.
...
+- Bump zendframework/zend-inputfilter from 2.8.1 to 2.8.2 - [#31431](https://github.com/owncloud/core/issues/31431)
+- Bump icewind/smb from 1.1.0 to 3.0.0 in /apps/files_external/3rdparty - [#31521](https://github.com/owncloud/core/issues/31521)
+- Bump symfony 3.4.9 to 3.4.11 - [#31571](https://github.com/owncloud/core/issues/31571)
 - Update jsdoc requirement to ~3.5.5 - [#30036](https://github.com/owncloud/core/issues/30036)
+- Removed example theme which now lives in the [theme-example repository](https://github.com/owncloud/theme-example) - [#31447](https://github.com/owncloud/core/issues/31447)
 
 ### Removed
 ### Fixed
+- Improve user-sync command help description - [#31691](https://github.com/owncloud/core/issues/31691)
+- Fix deletion of files in trashbin in some partial selection scenarios - [#31700](https://github.com/owncloud/core/issues/31700)
```
Comment (phil-davis) [inline CHANGELOG.md]: Also fixes restore of partial selection

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,17 +5,56 @@ All notable changes to this project will be documented in this file.
...
+- Exclude uploads directory from read-only cache mask, fixes guest app chunked uploads - [#31596](https://github.com/owncloud/core/issues/31596)
+- Properly normalize paths for event, no &$magic needed - [#31689](https://github.com/owncloud/core/issues/31689)
+- Use the correct user id in login related Symfony events - [#31605](https://github.com/owncloud/core/issues/31605)
+- Fix public link dialog issue when collaborative tags app is disabled - [#31581](https://github.com/owncloud/core/issues/31581)
+- Fix updating public link share in transfer ownership command - [#31176](https://github.com/owncloud/core/issues/31176)
+- Do not set the password again if it hasn't changed - [#31370](https://github.com/owncloud/core/issues/31370)
+- Use correct l10n to translate 'password was changed' email - [#31553](https://github.com/owncloud/core/issues/31553)
+- Improve text in settings/personal App Password - [#31539](https://github.com/owncloud/core/issues/31539)
+- Fix default langage code example - [#31448](https://github.com/owncloud/core/issues/31448)
```
Comment (phil-davis) [inline CHANGELOG.md]: s/langage/language

**Review by phil-davis — COMMENTED** [review]
Just little comments - text is generally fine.

**Comment by PVince81** [comment]
I've updated the changelog based on review comments and also added more of the stuff that got merged.
For the object store backport I rather mentionned the original master PRs as they had more information than the big backport PR.

@DeepDiver1975

File: CHANGELOG.md [hunk 3]
```diff
@@ -27,13 +32,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Bump symfony 3.4.9 to 3.4.11 - [#31571](https://github.com/owncloud/core/issues/31571)
 - Update jsdoc requirement to ~3.5.5 - [#30036](https://github.com/owncloud/core/issues/30036)
 - Removed example theme which now lives in the [theme-example repository](https://github.com/owncloud/theme-example) - [#31447](https://github.com/owncloud/core/issues/31447)
+- A user member of multiple groups is now excluded from sharing if at least one of their group is configured for exclusion - [#31737](https://github.com/owncloud/core/issues/31737)
```
Comment (phil-davis) [inline CHANGELOG.md]: A user who is a member of...

**Comment by PVince81** [comment]
squashed

File: CHANGELOG.md (hunk 3 above)
Comment (PVince81) [reply CHANGELOG.md]: fixed

**Pull request merged**